lib/internal/bootstrap_node.js

```diff
@@ -26,11 +26,6 @@
     setupProcessICUVersions();
 
     setupGlobalVariables();
-    const browserGlobals = !process._noBrowserGlobals;
-    if (browserGlobals) {
-      setupGlobalTimeouts();
-      setupGlobalConsole();
-    }
```

Comment:
It is desirable to load the console first for debugging reasons.

Reply:
Well that is a bit tricky if we want to instantiate it eagerly without the get call to do the Instantiation because `internal/process/stdio` has to be loaded first. Would it be fine to move the necessary part up that load `stdio` and keep the eager instantiation?

Reply:
Richard